### PR TITLE
Update the "Submitting a Patch to launchpad" How-To

### PR DESCRIPTION
**Pull request by RuinedYourLife**

### CODE REVIEW

**Review by ineesalmeida — COMMENTED** [review]
Changes look great! Left a couple of comments

File: how-to/contributing-changes.rst [hunk 1]
```diff
@@ -15,14 +15,28 @@ or on IRC (``#launchpad-dev`` on ``irc.libera.chat``).
...
 .. _launchpad-dev: https://launchpad.net/~launchpad-dev
 
-Get the Launchpad source
-----------------------------
+Fork the Launchpad repository
+-----------------------------
 
-Once you have decided on the change to be made, clone the repository.
+Navigate to the Launchpad project page.
+Click "Fork it to your account" to create a copy of the project in your
```
[the remark this answers is not in the document]
Comment (RuinedYourLife) [reply how-to/contributing-changes.rst:22]: added the link

File: how-to/contributing-changes.rst [hunk 2]
```diff
@@ -15,14 +15,28 @@ or on IRC (``#launchpad-dev`` on ``irc.libera.chat``).
...
+Launchpad account.
+
+Then, clone your fork of the repository:
+
+.. code-block:: bash
+
+    git clone git+ssh://~<your-username>@git.launchpad.net/~<your-username>/launchpad
```
[the remark this answers is not in the document]
Comment (RuinedYourLife) [reply how-to/contributing-changes.rst:29]: mentioned it's assumed to be done

File: .custom_wordlist.txt [hunk 3]
```diff
@@ -78,6 +78,8 @@ dbschema
 dbupgrade
 ddl
 DebuggingWithGdb
+deployable
```
[the remark this answers is not in the document]
Comment (RuinedYourLife) [reply .custom_wordlist.txt:81]: good catch! i forgot to remove these when working with an other file

[the remark this answers is not in the document]
Comment (ineesalmeida) [reply .custom_wordlist.txt:81]: There is still "deployable" and "buildbot" in this list that don't seem to have been added in this PR

**Review by ineesalmeida — APPROVED** [review]
Just left a couple of very minor comments, but :+1: on these changes!

File: .custom_wordlist.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (RuinedYourLife) [reply .custom_wordlist.txt]: idk why, but "buildbot" is used in some other places and it is showing up as added, while it's already there in the main repo?
deployable was used in urls so i thought it was needed but nevermid ^^'

**Pull request merged**